From 0b4b9aaa85c85c0326af039ce3dcb191af37b50b Mon Sep 17 00:00:00 2001
From: Xelo <williamchang435@gmail.com>
Date: Wed, 15 Sep 2021 06:21:59 +1000
Subject: [PATCH] Update Quaternion.mapping (#2662)

---
 .../net/minecraft/util/math/Quaternion.mapping     | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/mappings/net/minecraft/util/math/Quaternion.mapping b/mappings/net/minecraft/util/math/Quaternion.mapping
index 41a51066de..1b9a7c9b89 100644
--- a/mappings/net/minecraft/util/math/Quaternion.mapping
+++ b/mappings/net/minecraft/util/math/Quaternion.mapping
@@ -35,6 +35,20 @@ CLASS net/minecraft/class_1158 net/minecraft/util/math/Quaternion
 		ARG 2 y
 		ARG 3 z
 		ARG 4 w
+	METHOD method_35820 toEulerYxz ()Lnet/minecraft/class_1160;
+	METHOD method_35821 fromEulerYxz (FFF)Lnet/minecraft/class_1158;
+		ARG 0 x
+		ARG 1 y
+		ARG 2 z
+	METHOD method_35823 fromEulerXyzDegrees (Lnet/minecraft/class_1160;)Lnet/minecraft/class_1158;
+	METHOD method_35824 toEulerYxzDegrees ()Lnet/minecraft/class_1160;
+	METHOD method_35825 fromEulerXyz (FFF)Lnet/minecraft/class_1158;
+		ARG 0 x
+		ARG 1 y
+		ARG 2 z
+	METHOD method_35826 fromEulerXyz (Lnet/minecraft/class_1160;)Lnet/minecraft/class_1158;
+	METHOD method_35827 toEulerXyz ()Lnet/minecraft/class_1160;
+	METHOD method_35828 toEulerXyzDegrees ()Lnet/minecraft/class_1160;
 	METHOD method_4921 getX ()F
 	METHOD method_4922 getY ()F
 	METHOD method_4923 getZ ()F
-- 
2.32.0.windows.2

