From 587d189c77701605844145b260b521907c8e2edc Mon Sep 17 00:00:00 2001
From: NotSomeBot <43829743+mariusdkm@users.noreply.github.com>
Date: Wed, 20 Oct 2021 22:44:20 +0200
Subject: [PATCH] name function method_26318 applyMovementInput (#2773)

---
 mappings/net/minecraft/entity/LivingEntity.mapping | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/mappings/net/minecraft/entity/LivingEntity.mapping b/mappings/net/minecraft/entity/LivingEntity.mapping
index 4840cf485b..b5685bac10 100644
--- a/mappings/net/minecraft/entity/LivingEntity.mapping
+++ b/mappings/net/minecraft/entity/LivingEntity.mapping
@@ -215,6 +215,9 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 			COMMENT the source entity or {@code null} for non-entity sources
 	METHOD method_26084 (Lnet/minecraft/class_1309;)V
 		ARG 0 player
+	METHOD method_26318 applyMovementInput (Lnet/minecraft/class_243;F)Lnet/minecraft/class_243;
+		ARG 1 movementInput
+		ARG 2 slipperiness
 	METHOD method_26319 canWalkOnFluid (Lnet/minecraft/class_3611;)Z
 		ARG 1 fluid
 	METHOD method_26825 getAttributeValue (Lnet/minecraft/class_1320;)D
-- 
2.32.0.windows.2

